PolySTAR-Taproot-project/src/communication/cv_serial_data.hpp

```diff
@@ -0,0 +1,131 @@
+#ifndef CV_SERIAL_DATA_HPP_
+#define CV_SERIAL_DATA_HPP_
+
+#include "tap/communication/serial/ref_serial_data.hpp"
+#include "tap/util_macros.hpp"
+
+using tap::communication::serial::RefSerialData;
+using tap::communication::serial::Uart;
+
+namespace src::communication::cv
+{
+/**
+ * Contains enum and struct definitions used in the `CVSerial` and `CVHandler` classes
+ * 
+ */
+class CVSerialData
+{
+public:
+    class Tx
+    {
+    public:

```

Comment:
I'm more of a user of `enum class`, even more so if the enums are public. Though this is in its own namespace so that's probably fine